spectrocloud/resource_cluster_aws.go

```diff
@@ -84,6 +84,17 @@ func resourceClusterAws() *schema.Resource {
 				Description: "ID of the cloud config used for the cluster. This cloud config must be of type `azure`.",
 				Deprecated:  "This field is deprecated and will be removed in the future. Use `cloud_config` instead.",
 			},
+			"approve_cluster_repave": {
+				Type:        schema.TypeBool,
+				Default:     false,
+				Optional:    true,
+				Description: "To authorize the cluster repave, set the value to true for approval and false to decline. By default, it is set to true.",
+			},
+			"repave_state": {
```

Comment:
It doesn't seem that we need this in a state file. What is the use-case @SivaanandM ?

Reply:
Basically instead of get cluster in day 2 operation,  check the current repave_state set by read operations